internal/util/testutil/postgresql.go

```diff
@@ -72,7 +72,7 @@ func TestPostgreSQLURI(tb testtb.TB, ctx context.Context, baseURI string) string
 	_, err = p.Exec(ctx, q)
 	require.NoError(tb, err)
 
-	q = fmt.Sprintf("CREATE DATABASE %s", pgx.Identifier{name}.Sanitize())
+	q = fmt.Sprintf("CREATE DATABASE %s TEMPLATE template1", pgx.Identifier{name}.Sanitize())
```

Comment:
I'm just curious why you're adding `TEMPLATE template1` explicitly here given that template1 is the default template when creating a database in PostgreSQL.

Reply:
To make it explicit. The future PR will update `template` in `envtool`, so it would be easier to find the reference by searching